tests/integration_tests/scheduler/test_lsf_driver.py

```diff
@@ -93,6 +94,26 @@ async def finished(iens, returncode, aborted):
     assert aborted_called
 
 
+@pytest.mark.parametrize("runpath_supplied", [(True), (False)])
+async def test_lsf_info_file_written(runpath_supplied, tmp_path):
```

Comment:
I'd just change the name slightly :smile: 
`test_that_lsf_info_file_is_created`

Reply:
fixed